elide-async/src/main/java/com/yahoo/elide/async/service/DefaultAsyncQueryDAO.java

```diff
@@ -195,4 +199,30 @@ protected Object executeInTransaction(DataStore dataStore, Transactional action)
         }
         return result;
     }
+
+    @Override
+    @SuppressWarnings("unchecked")
+    public Collection<AsyncQuery> getActiveAsyncQueryCollection(String filterExpression) {
```

Comment:
The interface contract for the AsyncQueryDAO would be cleaner if the filterExpression for this method and others was an actual FilterExpression object instead of a String.